docs/manual/replacements_and_deletions.md

```diff
@@ -27,6 +27,23 @@ The following changes can only be rolled out through replacement:
 The number of inflight replacements can be configured by setting `maxConcurrentReplacements`, per default the operator will replace all misconfigured process groups.
 Depending on the cluster size this can require a quota that is has double the capacity of the actual required resources.
 
+## Using The Maintenance Mode
+
+The FoundationDB Kubernetes operator supports to make use of the [maintenance mode](https://github.com/apple/foundationdb/wiki/Maintenance-mode) in FoundationDB.
+Using the maintenance mode in FoundationDB will reduce the data distribution and disruption when Storage Pods must be updated.
+The following addition to the `FoundationDBCluster` resource will enable the maintenance mode for this cluster:
+
+```yaml
+spec:
+    automationOptions:
+      maintenanceModeOptions:
+        UseMaintenanceModeChecker: true
```

Comment:
Setting maintenance mode is done by running a command from fdbcli, so I am not sure why this needs to be added to the FoundationDBCluster spec in order to enable maintenance mode.

Reply:
That's right. This option is about the operator making use of the maintenance mode. Per default the operator will not use this option and just take down Pods if the Pod spec must be changed. If this setting is enabled the operator will make use of the maintenance mode before taking down the Pods.